src/events/deposits.controller.ts

```diff
@@ -55,6 +55,17 @@ export class DepositsController {
     return { address };
   }
 
+  @ApiOperation({
+    summary:
+      'The min and max increment of IRON allowed to be deposited to Iron Bank',
+  })
+  @Get('min_and_max_deposit')
+  minAndMaxDeposit(): { minDeposit: number; maxDeposit: number } {
+    const maxDeposit = this.configService.get<number>('MAX_DEPOSIT');
+    const minDeposit = this.configService.get<number>('MIN_DEPOSIT');
+    return { minDeposit, maxDeposit };
```

Comment:
We generally use snake case in returned fields from the API
```suggestion
    return { min_deposit, max_deposit };
```

Reply:
What do you think about `min_deposit_size` / `max_deposit_size` or `min_size` / `max_size`?